test/Broadway/Domain/DateTimeTest.php

```diff
@@ -28,12 +28,23 @@ public function it_converts_back_and_forth()
 
     /**
      * @test
+     * @dataProvider providesLocales
      */
-    public function it_creates_now()
+    public function it_creates_now($locale)
     {
+        $this->setLocale(LC_ALL, $locale);
```

Comment:
It's actually not LC_TIME that causes issue here, but LC_NUMERIC (float number format). I think it's best to keep LC_ALL.